package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "node-js-course",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
Rename to application entry point, please

FYI: `./dist/app.js`

Reply:
Done, thx